docs/howtos/layers/labels.md

```diff
@@ -79,67 +339,63 @@ nbscreenshot(viewer, alt_text="Segmentation of coins in an image, displayed usin
 viewer.close()
 ```
 
-## Arguments of `view_labels` and `add_labels`
+### Arguments of `view_labels` and `add_labels`
 
 {meth}`~napari.view_layers.view_labels` and {meth}`~napari.Viewer.add_labels`
 accept the same layer-creation parameters.
 
 ```{code-cell} python
-:tags: [hide-cell]
+:tags: [hide-output]
 
 help(napari.view_labels)
 ```
```

Comment:
Yes - and if you click the arrow it will show you the full output of the help on napari.view_labels. It was originally fully hidden which I think is even less useful 😅 But we can explore other options if you have suggestions

Reply:
I think i would either:
1) show output and say here's the blah blah
or 
2) say something like `you can see the full blah blah by using the following command`

Reply:
I made this change in codespace.